.github/workflows/pre-commit.yaml

```diff
@@ -2,9 +2,31 @@ name: pre-commit
 
 on:
   pull_request:
+  workflow_call:
+    inputs:
+      pre-commit-config:
+        required: true
+        type: string
+        default: .pre-commit-config.yaml
+    secrets:
+      token:
+        required: false
 
 jobs:
   pre-commit:
-    uses: autowarefoundation/autoware-github-actions/.github/workflows/reusable-pre-commit.yaml@tier4/proposal
-    with:
-      pre-commit-config: .pre-commit-config.yaml
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set pre-commit-config
+        run: |
+          PRE_COMMIT_CONFIG=${{ inputs.pre-commit-config }}
+          echo "PRE_COMMIT_CONFIG=${PRE_COMMIT_CONFIG:=.pre-commit-config.yaml}" >> $GITHUB_ENV
```

Comment:
Why did you add this section? I guess it's not necessary.

Reply:
Ah, it's because we explicitly use `--config`.

Reply:
Mm... should we make pre-commit a composite action? :thinking: 
I'll consider it.